src/Data/List.purs

```diff
@@ -632,18 +637,65 @@ tails list@(Cons _ tl)= list : tails tl
 --------------------------------------------------------------------------------
 
 -- | Remove duplicate elements from a list.
+-- | Keeps the first duplicates found and preserves ordering otherwise.
+-- |
+-- | ```purescript
+-- | nub 1:2:1:3:3:Nil = 1:2:3:Nil
```

Comment:
Todo - change these examples to use `==`. Didn't catch this when converting from the Array docs. Will need to fix this in Array as well.